app/src/main/res/values/oat_sms_texts.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="oat_sms_message_error_feature_disabled">OAT: \n The feature %s is disabled and cannot be accessed at the moment. Enable this feature in the settings of the App.\n IMPORTANT: Delete the message above to preserve your password!</string>
+    <string name="oat_sms_message_error_feature_not_found">OAT: \n The requested feature %s does not exist. Ensure that you typed the name correctly.\n IMPORTANT: Delete the message above to preserve your password!</string>
+    <string name="oat_sms_message_send_gps_position">OAT: \n The device is currently located at the following coordinates: %f,%f\n IMPORTANT: Delete the message above to preserve your password!</string>
+    <string name="oat_sms_message_error_invalid_password">OAT: \n The password \"%s\" is invalid. Ensure that you typed the name correctly.</string>
```

Comment:
Ouch. We shouldn't send the user his password even if it is wrong. And a password isn't a name. Please fix the typo(s)

Reply:
Correct, fixed